app/components/QrCodeScanner/QrCodeScanner.jsx

```diff
@@ -1,12 +1,23 @@
 // @flow
-import React, { Component } from 'react'
+import React, { Component, Fragment } from 'react'
 import Instascan from 'instascan'
 
+import Loading from '../../containers/App/Loading'
+
 type Props = {
-  callback: (content: string, scannerInstance: any) => any
+  callback: (content: string, scannerInstance: any) => any,
+  theme: string
```

Comment:
Whoops.. forgot to add `withThemeData` to `index.jsx`. Would force push but the GitHub comments I put in the code would get trampled :( I'll wait till you approve?

Reply:
Thats fine 👍